src/vendor/@carbon/colors/package.json

```diff
@@ -1,4 +1,11 @@
 {
+  "_args": [
+    [
+      "@carbon/colors@10.8.0",
+      "/Users/dusan.milko/Projects/dm/carbon-addons-angular"
```

Comment:
I don't think this is supposed to be there.

Reply:
updated